modules/core/src/lib/attribute.js

```diff
@@ -191,8 +190,6 @@ export default class LayerAttribute extends Attribute {
           state.needsRedraw = true;
         }
       }
-    } else {
-      state.isExternalBuffer = false;
```

Comment:
What happens when an attribute is switched from using external buffer to using own update function? This flag is never cleared.